### DISCUSSION TITLE
how to Database First, or import DBContext Entities into Sergen

### DISCUSSION THREAD

**Discussion by fasteddys** [serenity-is/Serenity#6515] [General]
Hello, I found your video very helpful. I had to watch it 3 times :) but after looking at your/volkan MSDN video and seeing the documentation, I am still seeking help, and not sure how to build code `using sergen` from `Database Tables` _**First**_.

- [ ] Can you please share some options on how to generate the sergen context from DB first
- [ ] Also how to import existing POCO's/Entities or another _preexisting_ `DBcontext codes/pocos` into sergen.

![image](https://user-images.githubusercontent.com/96149622/186504979-7ab2b6d6-4dc8-45ea-b523-c2bec39e96bc.png)

**Comment by VictorTomaili** [comment]
There is no sergen context in serenity,

you need to install `dotnet tool install sergen` (this tool automatically installed when you build a serene template project)
and use `dotnet sergen` to run scaffolding to create modules from the existing database.

**Reply by fasteddys** [reply]
Thanks
